scripts/boot-p2p.py

```diff
@@ -17,184 +17,180 @@
 import time
 import sys
 import random
+import collections
 
+parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
-parser = argparse.ArgumentParser(
-             formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-parser.add_argument("-b", "--boot",
-                    action='store_true',
-                    help="boot network by creating resources and starting services by network name")
-parser.add_argument("--bootstrap-command",
-                    dest='bootstrap_command',
-                    type=str,
-                    default="run --port 40400 --standalone",
-                    help="bootstrap container run command")
 parser.add_argument("-c", "--cpuset-cpus",
                     dest='cpuset_cpus',
                     type=str,
                     default="0",
                     help="set docker cpuset-cpus for all nodes. Allows limiting execution in specific CPUs")
+
 parser.add_argument("-i", "--image",
                     dest='image',
                     type=str,
                     default="coop.rchain/rnode:latest",
                     help="source repo for docker image")
+
 parser.add_argument("-m", "--memory",
                     dest='memory',
                     type=str,
                     default="2048m",
                     help="set docker memory limit for all nodes")
+
 parser.add_argument("-n", "--network",
                     dest='network',
                     type=str,
                     default="rchain.coop",
                     help="set docker network name")
+
 parser.add_argument("-p", "--peers-amount",
                     dest='peer_amount',
                     type=int,
                     default="2",
                     help="set total amount of peers for network")
-parser.add_argument("--rnode-directory",
-                    dest='rnode_directory',
-                    type=str,
-                    default="/var/lib/rnode",
-                    help="rnode container root directory on each container")
+
 parser.add_argument("-r", "--remove",
                     action='store_true',
                     help="forcibly remove containers that start with bootstrap and peer associated to network name")
-parser.add_argument("-t", "--thread-pool-size",
-                    dest='thread_pool_size',
+
+parser.add_argument("--bonds",
+                    dest="bonds_file",
+                    type=str,
+                    default="bonds_file",
+                    help="set the bond file of the genesis process")
+
+parser.add_argument("--wallet",
+                    dest="wallet_file",
+                    type=str,
+                    default="/var/lib/rnode/wallets.txt",
```

Comment:
This default value won't be valid on my machine

Reply:
Agreed. Probably better that there is not a default since it's not a required flag.